BraveRewardsUI/BATValue.swift

```diff
@@ -38,7 +38,7 @@ public struct BATValue: DisplayableOption {
   }
   
   public var displayString: String {
-    return String(format: "%.1f", value)
+    return String(format: "%.3f", value)
```

Comment:
I know we're having this conversation already but just to document: Are we sure we want to always show 3 decimal places instead of a min/max number of decimals shown based on the value itself? We can easily do this with `NumberFormatter`

Reply:
^^^ @jsecretan

Reply:
As we always show 3 decimal places for desktop, I will merge as-is. And if a decision is made to change we should raise a new issue for all platforms